### ISSUE TITLE
Change image sizes

### ISSUE THREAD

**Issue by mdefende** [uabrc/uabrc.github.io#202]
From a style perspective, most images in our docs completely overwhelm the surrounding text making navigation and comprehension more difficult. Currently, images are either fit to the screen width or render at their native resolution. We should shrink the images that fill the entire screen. We can do this using the style attribute from the attr_list extension which we have enabled. To change the size of an image to have a 100 px width, it would look like this:

`![! alt text ><](image.png){: style="width:100px}`

Crucially, this does not change the image size for the lightbox functionality, it will still render at full size, so for people who can't fully see the image directly on the page, they will be able to see it at full resolution in the lightbox.

**Comment by mdefende** [comment]
Yeah, setting the width as a % references the width of the screen, not the original width of the image. So the smaller images we have get blown up if we set a global value of X%. But setting an absolute pixel value will behave differently based on screen resolution I believe. So like William said, if there's a way to choose the minimum of either the image size or the X% size, that would be more ideal for a global option, then we can manually change images as necessary

**Comment by mdefende** [comment]
Also, I've gone through and done it for all of the images in the Cloud section in a pull request I'm going to submit today. We can change it to a global option later if we figure that out.

**Comment by jgordini** [comment]
Typically you setup media queries for each screen size.
then nest your class definitions inside it.
Example.

// default
.lightbox img {
                    max-width: 100%;
}

// desktops
@media only screen and (min-width : 992px) {
                    .lightbox img {
                    Max-width:25em;
    }
}

https://stackoverflow.com/questions/6370690/media-queries-how-to-target-desktop-tablet-and-mobile



--
Jeremy Gordinier | Web Designer
Information Technology
UAB | The University of Alabama at Birmingham
P: 205-549-1575 | ***@***.******@***.***>
Pronouns<https://www.uab.edu/dei/institutional-equity/inclusive-language-guide>: He, Him, His
[A picture containing graphical user interface  Description automatically generated]<https://www.uab.edu/it/home/>


From: mdefende ***@***.***>
Date: Thursday, April 28, 2022 at 11:59 AM
To: uabrc/uabrc.github.io ***@***.***>
Cc: Gordinier, Jeremy ***@***.***>, Comment ***@***.***>
Subject: Re: [uabrc/uabrc.github.io] Change image sizes (Issue #202)

Yeah, setting the width as a % references the width of the screen, not the original width of the image. So the smaller images we have get blown up if we set a global value of X%. But setting an absolute pixel value will behave differently based on screen resolution I believe. So like William said, if there's a way to choose the minimum of either the image size or the X% size, that would be more ideal for a global option, then we can manually change images as necessary

**Comment by jgordini** [comment]
Currently this is being overridden by your inline tag

img[alt$="><"] {
    display: block;
    max-width: 100%;
    height: auto;
    margin: 1.5rem;
    float: none!important;
}

--
Jeremy Gordinier | Web Designer
Information Technology
UAB | The University of Alabama at Birmingham
P: 205-549-1575 | ***@***.******@***.***>
Pronouns<https://www.uab.edu/dei/institutional-equity/inclusive-language-guide>: He, Him, His
[A picture containing graphical user interface  Description automatically generated]<https://www.uab.edu/it/home/>


From: Gordinier, Jeremy ***@***.***>
Date: Thursday, April 28, 2022 at 12:05 PM
To: uabrc/uabrc.github.io ***@***.***>, uabrc/uabrc.github.io ***@***.***>
Cc: Comment ***@***.***>
Subject: Re: [uabrc/uabrc.github.io] Change image sizes (Issue #202)
Typically you setup media queries for each screen size.
then nest your class definitions inside it.
Example.

// default
.lightbox img {
                    max-width: 100%;
}

// desktops
@media only screen and (min-width : 992px) {
                    .lightbox img {
                    Max-width:25em;
    }
}

https://stackoverflow.com/questions/6370690/media-queries-how-to-target-desktop-tablet-and-mobile



--
Jeremy Gordinier | Web Designer
Information Technology
UAB | The University of Alabama at Birmingham
P: 205-549-1575 | ***@***.******@***.***>
Pronouns<https://www.uab.edu/dei/institutional-equity/inclusive-language-guide>: He, Him, His
[A picture containing graphical user interface  Description automatically generated]<https://www.uab.edu/it/home/>


From: mdefende ***@***.***>
Date: Thursday, April 28, 2022 at 11:59 AM
To: uabrc/uabrc.github.io ***@***.***>
Cc: Gordinier, Jeremy ***@***.***>, Comment ***@***.***>
Subject: Re: [uabrc/uabrc.github.io] Change image sizes (Issue #202)

Yeah, setting the width as a % references the width of the screen, not the original width of the image. So the smaller images we have get blown up if we set a global value of X%. But setting an absolute pixel value will behave differently based on screen resolution I believe. So like William said, if there's a way to choose the minimum of either the image size or the X% size, that would be more ideal for a global option, then we can manually change images as necessary

**Comment by jgordini** [comment]
Let me test this on my fork and I will update this issue.


--
Jeremy Gordinier | Web Designer
Information Technology
UAB | The University of Alabama at Birmingham
P: 205-549-1575 | ***@***.******@***.***>
Pronouns<https://www.uab.edu/dei/institutional-equity/inclusive-language-guide>: He, Him, His
[A picture containing graphical user interface  Description automatically generated]<https://www.uab.edu/it/home/>


From: Gordinier, Jeremy ***@***.***>
Date: Thursday, April 28, 2022 at 12:07 PM
To: uabrc/uabrc.github.io ***@***.***>, uabrc/uabrc.github.io ***@***.***>
Cc: Comment ***@***.***>
Subject: Re: [uabrc/uabrc.github.io] Change image sizes (Issue #202)
Currently this is being overridden by your inline tag

img[alt$="><"] {
    display: block;
    max-width: 100%;
    height: auto;
    margin: 1.5rem;
    float: none!important;
}

--
Jeremy Gordinier | Web Designer
Information Technology
UAB | The University of Alabama at Birmingham
P: 205-549-1575 | ***@***.******@***.***>
Pronouns<https://www.uab.edu/dei/institutional-equity/inclusive-language-guide>: He, Him, His
[A picture containing graphical user interface  Description automatically generated]<https://www.uab.edu/it/home/>


From: Gordinier, Jeremy ***@***.***>
Date: Thursday, April 28, 2022 at 12:05 PM
To: uabrc/uabrc.github.io ***@***.***>, uabrc/uabrc.github.io ***@***.***>
Cc: Comment ***@***.***>
Subject: Re: [uabrc/uabrc.github.io] Change image sizes (Issue #202)
Typically you setup media queries for each screen size.
then nest your class definitions inside it.
Example.

// default
.lightbox img {
                    max-width: 100%;
}

// desktops
@media only screen and (min-width : 992px) {
                    .lightbox img {
                    Max-width:25em;
    }
}

https://stackoverflow.com/questions/6370690/media-queries-how-to-target-desktop-tablet-and-mobile



--
Jeremy Gordinier | Web Designer
Information Technology
UAB | The University of Alabama at Birmingham
P: 205-549-1575 | ***@***.******@***.***>
Pronouns<https://www.uab.edu/dei/institutional-equity/inclusive-language-guide>: He, Him, His
[A picture containing graphical user interface  Description automatically generated]<https://www.uab.edu/it/home/>


From: mdefende ***@***.***>
Date: Thursday, April 28, 2022 at 11:59 AM
To: uabrc/uabrc.github.io ***@***.***>
Cc: Gordinier, Jeremy ***@***.***>, Comment ***@***.***>
Subject: Re: [uabrc/uabrc.github.io] Change image sizes (Issue #202)

Yeah, setting the width as a % references the width of the screen, not the original width of the image. So the smaller images we have get blown up if we set a global value of X%. But setting an absolute pixel value will behave differently based on screen resolution I believe. So like William said, if there's a way to choose the minimum of either the image size or the X% size, that would be more ideal for a global option, then we can manually change images as necessary

**Comment by jgordini** [comment]
Adding the bottom declarations to extra.css and removing img[alt$="><"] should fix it.

This is just for my notes. Will test.

img[alt$="><"] {
    display: block;
    max-width: 100%;
    height: auto;
    margin: auto;
    float: none!important;
}


// adds a flexbox to the surrounding div
div.lightgallery {
    display: flex;
    justify-content: center;
}

// sizing the images down on desktops
@media only screen and (min-width : 992px) {
                    .lightgallery img {
       width: 75vw
            }
}

//adding the drop shadow and keeping the images at 100 viewport width for smaller screen sizes
.lightgallery img {
    width: 100vw;
         box-shadow: 5px 5px 13px rgba(0, 0, 0, 0.08);
}



--
Jeremy Gordinier | Web Designer
Information Technology
UAB | The University of Alabama at Birmingham
P: 205-549-1575 | ***@***.******@***.***>
Pronouns<https://www.uab.edu/dei/institutional-equity/inclusive-language-guide>: He, Him, His
[A picture containing graphical user interface  Description automatically generated]<https://www.uab.edu/it/home/>


From: mdefende ***@***.***>
Date: Friday, April 29, 2022 at 11:20 AM
To: uabrc/uabrc.github.io ***@***.***>
Cc: Gordinier, Jeremy ***@***.***>, Comment ***@***.***>
Subject: Re: [uabrc/uabrc.github.io] Change image sizes (Issue #202)

We only need the inline tag there to easily center the images. I didn't know how to do it but found that hack somewhere on stackoverflow. If you know of a better way to have images automatically center while implementing the sizing changes, that would be great

**Comment by jgordini** [comment]
I have added the drop shadow, mobile color fix and image sizing to my fork. You can see it in action here. 

https://jgordini.github.io/uabrc.github.io/uab_cloud/network_setup_basic/

If this is acceptable I will make a pull request.

**Comment by mdefende** [comment]
This looks great! Thank you a ton for working on it!

**Comment by jgordini** [comment]
Absolutely. It does remove the >< centering option and just centers everything by default. Because of CSS specificty it is difficult to implement both workflows. I may be able to do this but it would take a bit of time to figure out I think. I can push the changes for now and see if I can find an alternative solution.

**Comment by mdefende** [comment]
Hey @jgordini is there a way to change this to where smaller images are printed at their native resolution? All of the images are being printed at the same size which is great for some of the very large images, but not for the smaller ones. We can manually change them using the tags from above when necessary, but I was just wondering if there was an easier way to implement it in CSS

![image](https://user-images.githubusercontent.com/20781527/166473659-8e5d047e-b847-4ccc-b6be-a1800b0d2231.png)

**Comment by jgordini** [comment]
I think so. Let me look into it.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by jgordini** [comment]
had width instead of max-width in there. 
should be working now. 

you can see the update here. 

https://jgordini.github.io/uabrc.github.io/data_management/transfer/globus/

Old version
https://uabrc.github.io/data_management/transfer/globus/

**Comment by mdefende** [comment]
Bless, this is wonderful. Looks good to me

**Comment by jgordini** [comment]
I apologize I'm seeing some weird stuff in extra.css on my end. 
There's a Head section 
<<<<<<Head
a bunch of old code
and at the bottom 
>>>>>>> ca7117e354653a2a86a7d96fa9926c3abeb9053b

I'm assuming this is from an old merge. I can delete it but not sure if this is the way to fix it. 
you can see it here. 
https://jgordini.github.io/uabrc.github.io/stylesheets/extra.css
